### DISCUSSION TITLE
Support CSS zoom

### DISCUSSION THREAD

**Discussion by robertmylne** [tailwindlabs/tailwindcss#14288] [Ideas]
Support css zoom. This is a very effective property for scaling iframe's used for things such as previews.

Example: `zoom: 0.6`

**Comment by danilofuchs** [comment]
I am using `zoom: 0.75` for rendering an HTML e-mail inside an iframe

Firefox recently implemented Zoom so it's now Baseline

https://caniuse.com/?search=zoom
